Blender-AR-Project/AR-Test-Project.py

```diff
@@ -0,0 +1,369 @@
+                        cam.location = unity_vector_to_blender(words[1], words[2], words[3])
+                        cam.keyframe_insert('location')
+                    if include_camera_rotation:
+                        q = unity_quaternion_to_blender(words[7], words[4], words[5], words[6])
+                        cam.rotation_quaternion = q
+                        cam.keyframe_insert('rotation_quaternion')
+
+            if words[0] == 'd' and include_cloud:
+                id = int(words[1])
+                confidence = float(words[5])
+                if confidence >= 0.5:
+                    if not id in cloud_points:
+                        v = unity_vector_to_blender(words[2], words[3], words[4])
+                        cloud_points[id] = point_cloud_bmesh.verts.new(v)
+                    else:
+                        vert = cloud_points[id]
+                        vert.co = unity_vector_to_blender(words[2], words[3], words[4])
+
+            if words[0] == 'p' and include_planes:
+                complete_plane_bmesh()
+
+                id = words[1]
+                if not id in planes:
+                    plane_mesh = bpy.data.meshes.new("plane")
+                    plane = bpy.data.objects.new("Plane", plane_mesh)
+                    plane.rotation_mode = 'QUATERNION'
+                    scene.collection.objects.link(plane)
+                    planes[id] = plane
+                    plane_bm = bmesh.new()
+                else:
+                    plane = planes[id]
+                    plane_mesh = plane.data
+                    plane_bm = bmesh.new()
+                    plane_bm.from_mesh(plane_mesh)
+                    bmesh.ops.delete(plane_bm, geom=plane_bm.verts, context='VERTS')
+
+                plane.location = unity_vector_to_blender(words[2], words[3], words[4])
+                q = unity_quaternion_to_blender(words[8], words[5], words[6], words[7])
+                plane.rotation_quaternion = q @ ROTATE_X_90
+
+            if words[0] == 'b' and include_planes:
+                plane_bm.verts.new((float(words[1]), -float(words[2]), 0.0))
+
+    if include_cloud:
+        point_cloud_bmesh.to_mesh(point_cloud_mesh)
+        point_cloud_bmesh.free()
+    if include_planes:
+        complete_plane_bmesh()
+    
+    return {'FINISHED'}
+
+
+def unity_quaternion_to_blender(w, x, y, z):
+    return ROTATE_X_90 @ mathutils.Quaternion((-float(w), float(x), float(y), -float(z)))
+
+class AR_PT_import_main(bpy.types.Panel):
+    bl_space_type = 'FILE_BROWSER'
+    bl_region_type = 'TOOL_PROPS'
+    bl_label = ""
+    bl_parent_id = "FILE_PT_operator"
+    bl_options = {'HIDE_HEADER'}
+    
+    @classmethod
+    def poll(cls, context):
+        sfile = context.space_data
+        operator = sfile.active_operator
+        
+        return operator.bl_idname == "IMPORT_ANIM_OT_ar"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
```

Comment:
use_property_split in side with use_property_decorate to True, marks down the animation button  is the split makes room for it, and the decorate draws it. please add this